src/objc/CMakeLists.txt

```diff
@@ -15,8 +15,25 @@ if(APPLE AND HAS_MLCUSTOMMODEL)
     DEAD_STRIP
   )
 
+  find_library(ACCELERATE NAMES Accelerate)
```

Comment:
Since we sometimes share frameworks between different libraries within turicreate, I started moving these to the top level CMakeLists.txt. I think there may be one there already for `Accelerate`.

Reply:
Thanks for letting me know. It's even named exactly the same thing. Done.